src/_pytest/doctest.py

```diff
@@ -246,8 +246,9 @@ def __init__(
         parent: "Union[DoctestTextfile, DoctestModule]",
         runner: Optional["doctest.DocTestRunner"] = None,
         dtest: Optional["doctest.DocTest"] = None,
+        **kw,
```

Comment:
this should be `**kw: Any` otherwise it turns off type checking on this function

Reply:
Agree with `**kw: Any`, but just one missing type annotation doesn't turn off type checking for the function does it?